OnLand/ControlLaptop/static/assets/js/FlightProfile.js

```diff
@@ -13,11 +13,52 @@ function showSuccessConfigurationUpdate(from, align){
 	});
 }
 
+function validConfigFormValues() {
+  let formHasError = true;
```

Comment:
I dont think this variable name makes sense. formaHasError should start false and then flip when an error is found. But you want to return true for a good form. So maybe let noErrors = True?